### PR TITLE
add a buildkite job for allsky radiation with regression test

### PR DESCRIPTION
**Pull request by szy21**
Adds a buildkite job that uses allsky radiation and an entry into the regression tests. This helps determine if the allsky radiation results is deterministic.

### CODE REVIEW

**Comment by szy21** [comment]
bors try

**Comment by charleskawczynski** [comment]
We just need an mse table entry:
```
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"] = OrderedCollections.OrderedDict()
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:c, :ρ)] = 0.0
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:c, :ρe_tot)] = 0.0
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:c, :uₕ, :components, :data, 1)] = 0.0
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:c, :uₕ, :components, :data, 2)] = 0.0
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:c, :ρq_tot)] = 0.0
all_best_mse["sphere_aquaplanet_rhoe_equilmoist_allsky"][(:f, :w, :components, :data, 1)] = 0.0
```
This will still error, FYI, though. The one additional thing needed is to increment the ref counter and reset all other tables to zero.

Here, I'll push a commit.

**Review by charleskawczynski — APPROVED** [review]

**Comment by charleskawczynski** [comment]
Oof, yeah, this still seems to be non-deterministic

**Comment by szy21** [comment]
That's strange, I'll look into it.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -269,6 +269,10 @@ steps:
       - label: ":computer: aquaplanet (ρe_tot) equilmoist clearsky radiation"
         command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --surface_scheme bulk --moist equil --rad clearsky --microphy 0M --job_id sphere_aquaplanet_rhoe_equilmoist_clearsky --dt 400secs --t_end 2.5days"
         artifact_paths: "sphere_aquaplanet_rhoe_equilmoist_clearsky/*"
+      
+      - label: ":computer: aquaplanet (ρe_tot) equilmoist allsky radiation"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --rayleigh_sponge true --job_id sphere_aquaplanet_rhoe_equilmoist_allsky --dt 200secs --t_end 2days --regression_test true"
```
Comment (charleskawczynski) [inline .buildkite/pipeline.yml:274]:
```suggestion
        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --enable_threading false --vert_diff true --surface_scheme bulk --moist equil --rad allskywithclear --microphy 0M --rayleigh_sponge true --job_id sphere_aquaplanet_rhoe_equilmoist_allsky --dt 200secs --t_end 2days --regression_test true"
```

**Comment by szy21** [comment]
bors try

**Comment by szy21** [comment]
bors try

**Comment by szy21** [comment]
bors try

**Pull request closed without merging**